Lab 2

d3nik opened the pull request.

d3nik committed:
Lab.2 patch 1.0

- portfolio.html, folder css and style.css created;
- portfolio.html filled with content;
- nav href edited.

d3nik committed:
Lab.2 patch 1.1

– Images added;
– Connected some fonts;
– Some class added in index.html and portfolio.html;
– Some general changes in styles.css.

vozniuk-lntu reviewed (changes requested):
1. колір посилань має бути сірий https://prnt.sc/is-d5bu1LQ9O
2. проставити letter-spacing 3% = 0.03em
3. h1 в intro зробити капсом
4. кнопки не успадковують шрифт https://prnt.sc/GjGD7gpTCbVC
5. не такий розмір шрифта https://prnt.sc/mwpKP4dEnAO5
6. там шрифт звичайний 400 https://prnt.sc/zQRnyRHjfj9C
7. в портфоліо одна секція з кнопками на карточками
8. описи картинок укр а не англ
9. глянь розмір шрифта https://prnt.sc/vHo-E1PjwVSj
10. body {
    font-family: "Roboto", "Raleway", sans-serif; Raleway тобі не потрібний

vozniuk-lntu reviewed (approved):
виправ з третій

d3nik committed:
Lab.2 fix-patch 1.2

– link-color changed in header;
– letter-spacing added to every text;
– intro.h1 in uppercase now;
– button have font-family;
– fixed font-size;
– filters button now in section with cards;
– some other fixes;
